canopeum_frontend/src/constants/weatherImageMap.ts

```diff
@@ -1,24 +1,21 @@
-import { readonlyURL } from 'readonly-types'
+import { type ReadonlyURL, readonlyURL } from 'readonly-types'
 
 type WMOCategoriesImageUrls = {
-  readonly imageUrl: string | undefined,
+  readonly imageUrl: ReadonlyURL | undefined,
   WMOCategories: string[],
 }
 
-const getURLString = (internalUrl: string): string | undefined =>
-  readonlyURL(internalUrl, import.meta.url)?.href
-
 const WMOCategoriesImages: WMOCategoriesImageUrls[] = [
   {
-    imageUrl: '@assets/images/weather/sunny-bg.jpeg',
+    imageUrl: readonlyURL('../assets/images/weather/sunny-bg.jpeg', import.meta.url),
     WMOCategories: ['Clear sky', 'Mainly clear'],
   },
   {
-    imageUrl: getURLString('@assets/images/weather/cloudy-bg.jpeg'),
+    imageUrl: readonlyURL('../assets/images/weather/cloudy-bg.jpeg', import.meta.url),
```

Comment:
Can we no use webpack/ts-paths style `@` isntead of relative paths ?

Reply:
Unfortunaly it didn;t with relative path :(